### DISCUSSION TITLE
Taskbar icon labels

### DISCUSSION THREAD

**Discussion by Turtl31** [valinet/ExplorerPatcher#1558] [Ideas]
When the style is set to Windows 10 and the taskbar position is set to top (idk for other positions)
the program icons have labels (example: when I turn on Spotify in the taskbar it adds a label that says what song I am listening  to)
The taskbar should just show that Spotify is on and that's it like on windows 10

**Comment by Poopooracoocoo** [comment]
If this isn't what's happening already, what're you seeing instead?
